Gaijinizer patch/Map352.txt

```diff
@@ -160,10 +159,10 @@ Needless to say, it's a subtle difficulty...#GT
 
 ほんとバカにしてる……
 > CONTEXT: Map352/events/20/pages/0/20/Dialogue/enemy
-Moreover, that rock, despite the heavy noise
-weighs as much as Styrofoam...
+And even though those rocks sound like they're 
+heavy, they're as light as styrofoam.
 
-I'm really stupid...#GT
+They're just making fun of me...#Ron
 > END STRING
```

Comment:
Is this "me", or "us"?

Reply:
I figured that since she's the adult in the situation, and that she's also the fake Lampas sent to mess with Kazuya, that maybe she was complaining that they were unnecessarily drawing everything out with unnecessary and easy puzzles. But I dunno.

Reply:
Makes enough sense to me.